ceph-volume-nightly/build/build

```diff
@@ -14,11 +14,7 @@ update_vagrant_boxes
 
 cd src/ceph-volume/ceph_volume/tests/functional/$SUBCOMMAND
 
-if [ "$CEPH_BRANCH" = "mimic" ]; then
-    CEPH_ANSIBLE_BRANCH="stable-3.2"
-elif [ "$CEPH_BRANCH" = "luminous" ]; then
-    CEPH_ANSIBLE_BRANCH="stable-3.2"
-elif [ "$CEPH_BRANCH" = "nautilus" ]; then
+if [ "$CEPH_BRANCH" = "nautilus" ]; then
```

Comment:
@dsavineau 
I know it's not really related to the original intent of this PR but, aren't we missing the following code?

```bash
elif [ "$CEPH_BRANCH" = "octopus" ]; then
    CEPH_ANSIBLE_BRANCH="stable-5.0"
elif [ "$CEPH_BRANCH" = "pacific" ]; then
    CEPH_ANSIBLE_BRANCH="stable-6.0"
....
```

Reply:
I was thinking about this too but there's currently no nightly jobs for octopus or pacific so this can be done in a dedicated PR